server/store/datastore/migration/008_repo_add_remote_id.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Woodpecker Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package migration
+
+import (
+	"github.com/woodpecker-ci/woodpecker/server/model"
+	"xorm.io/xorm"
+)
+
+type RepoV008 struct {
+	RemoteID string `json:"-" xorm:"'remote_id'"`
```

Comment:
migration not needed, will be executed anyway ...!

https://woodpecker-ci.org/docs/development/guides#add-a-new-migration

Reply:
But it will fail if any future migration uses this field and upgrade is happening from version where this field is not added yet

Reply:
no it wont have a problem either ... if field is realy needed, because of a complex transformation then a sync2 with onld struct should be added there ... migrations are handled theme based not version based (that makes them backportable)

Reply:
woodpecker migrations != gitea migrations